architecture/nvidia-gpu-architecture-overview.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id="nvidia-gpu-architecture-overview"]
+= NVIDIA GPU architecture overview
+include::_attributes/common-attributes.adoc[]
+:context: nvidia-gpu-architecture-overview
+
+toc::[]
+
+NVIDIA supports using GPU resources on {product-title}.
+{product-title} is a security-centric and enterprise-grade hardened Kubernetes platform for
+deploying and managing Kubernetes clusters at scale, developed and supported by Red Hat.
+{product-title} includes enhancements to Kubernetes so users can easily configure and use
+NVIDIA graphics processing unit (GPU) resources for accelerating workloads.
+
+The NVIDIA GPU Operator uses the Operator framework within {product-title} to automate the management of all
+NVIDIA software components needed to provision GPU. These components include the NVIDIA drivers
+(to enable CUDA), Kubernetes device plugin for GPUs,
+the NVIDIA Container Toolkit,
+automatic node labeling using GPU feature discovery (GFD), DCGM based monitoring, and others.
+
```

Comment:
@egallen I'm not entirely satisfied with this intro text.

Reply:
@StephenJamesSmith What is the status of this comment?